IPython/terminal/embed.py

```diff
@@ -19,7 +19,7 @@
 from traitlets import Bool, CBool, Unicode
 from IPython.utils.io import ask_yes_no
 
-class KillEmbeded(Exception):pass
+class KillEmbedded(Exception):pass
```

Comment:
In this specific case, I think this is OK, but in general we should be very careful about renaming exposed objects, because it would break any code using them. I don't think this is likely to be used outside our code, though.